jbpm-console-ng-human-tasks/jbpm-console-ng-human-tasks-client/src/main/java/org/jbpm/console/ng/ht/client/editors/taskslist/grid/dash/DataSetTasksListGridPresenter.java

```diff
@@ -169,14 +169,16 @@ public void getData( final Range visibleRange ) {
                             currentTableSettings.getDataSetLookup().addOperation(filter);
                         }
                     }
-                    if( currentTableSettings.getDataSetLookup().getDataSetUUID().equals(HUMAN_TASKS_WITH_ADMIN_DATASET) ||
-                        currentTableSettings.getDataSetLookup().getDataSetUUID().equals(HUMAN_TASKS_WITH_USER_DATASET)) {
+                    boolean isAdmindDataset = currentTableSettings.getDataSetLookup().getDataSetUUID().equals(HUMAN_TASKS_WITH_ADMIN_DATASET);
 
+                    if( isAdmindDataset ||
+                        currentTableSettings.getDataSetLookup().getDataSetUUID().equals(HUMAN_TASKS_WITH_USER_DATASET)) {
                         if (currentTableSettings.getDataSetLookup().getFirstFilterOp() != null) {
-                            currentTableSettings.getDataSetLookup().getFirstFilterOp().addFilterColumn(getUserGroupFilters()); // add actualowner = idenfier or in my groups
+                            // add  logged user is owner or task in my groups
```

Comment:
Can't say this comment helps me understand what's going on. Could this be refined? Potentially even skipped as the code is clear enough.